usermanager/src/main/java/org/pesmypetcare/usermanager/clients/pet/PetManagerClient.java

```diff
@@ -0,0 +1,480 @@
+        }
+        String jsonArray = response.substring(1, response.length() - 1);
+        String[] pets = jsonArray.split(",\\{");
+        List<Pet> petsList = new ArrayList<>();
+        petsList.add(GSON.fromJson(pets[0], Pet.class));
+        for (int i = 1; i < pets.length; i++) {
+            pets[i] = "{" + pets[i];
+            petsList.add(GSON.fromJson(pets[i], Pet.class));
+        }
+        return petsList;
+    }
+
+    /**
+     * Deletes a pet of the specified user.
+     *
+     * @param accessToken The personal access token for the account
+     * @param username The user's username
+     * @param petName The pet's name
+     * @return The response code
+     * @throws ExecutionException When the retrieval of the pets fails
+     * @throws InterruptedException When the retrieval is interrupted
+     */
+    public int deletePet(String accessToken, String username, String petName)
+            throws ExecutionException, InterruptedException {
+        taskManager = taskManager.resetTaskManager();
+        taskManager.setTaskId(DELETE);
+        StringBuilder response = taskManager.execute(BASE_URL + PETS_PATH + username + SLASH + petName, accessToken)
+                .get();
+        return Integer.parseInt(response.toString());
+    }
+
+    /**
+     * Deletes all pets of the specified user.
+     *
+     * @param accessToken The personal access token for the account
+     * @param username The user's username
+     * @return The response code
+     * @throws ExecutionException When the retrieval of the pets fails
+     * @throws InterruptedException When the retrieval is interrupted
+     */
+    public int deleteAllPets(String accessToken, String username) throws ExecutionException, InterruptedException {
+        taskManager = taskManager.resetTaskManager();
+        taskManager.setTaskId(DELETE);
+        StringBuilder response = taskManager.execute(BASE_URL + PETS_PATH + username, accessToken).get();
+        return Integer.parseInt(response.toString());
+    }
+
+    /**
+     * Gets the value for the specified field of the pet on the database.
+     *
+     * @param accessToken The personal access token for the account
+     * @param username The pet's owner username
+     * @param petName The pet's name
+     * @param field Name of the field to retrieve the value from
+     * @return The value from the field on the database
+     * @throws ExecutionException When the retrieval fails
+     * @throws InterruptedException When the retrieval is interrupted
+     */
+    public Object getSimpleField(String accessToken, String username, String petName, String field)
+            throws ExecutionException, InterruptedException {
+        PetData.checkSimpleField(field);
+        taskManager = taskManager.resetTaskManager();
+        taskManager.setTaskId(GET);
+        StringBuilder response = taskManager
+                .execute(BASE_URL + PETS_PATH + username + SLASH + petName + "/simple/" + field, accessToken).get();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "/simple/" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5561038)